src_rebuild/Game/C/wheelforces.c

```diff
@@ -528,6 +528,32 @@ void AddWheelForcesDriver1(CAR_DATA* cp, CAR_LOCALS* cl)
 	else
 	{
 		cp->hd.wheel_speed = cdz / 64 * (cl->vel[2] / 64) + cdx / 64 * (cl->vel[0] / 64);
+
+		// [A] wibbly wobbly fuckery hack...
+		if (car_cos->extraInfo & 4)
+		{
+			if (cp->thrust == 0 && cp->handbrake && (cp->hd.speed > 0 && cp->hd.speed < 4))
+			{
+				cp->hd.speed--;
+				cp->hd.wheel_speed >>= 1;
+
+				cp->hd.acc[0] >>= 1;
+				cp->hd.acc[1] >>= 1;
+				cp->hd.acc[2] >>= 1;
+
+				cp->hd.aacc[0] >>= 2;
+				cp->hd.aacc[1] >>= 2;
+				cp->hd.aacc[2] >>= 2;
+
+				//cp->st.n.linearVelocity[0] >>= 1;
```

Comment:
These are commented out because otherwise the car will come to a _screeching_ halt and basically act like dead weight.